### PR TITLE
chore(cleanup): remove unused markup

### PR DESCRIPTION
**Pull request by josephmcg**
**What this PR does** 📖
- remove leftover interval code
- switch some code to shorthand
- remove unused case since all image messages are encoded as 'file'. I removed all of the other associated code a week or two ago, missed this one bit

**Which issue(s) this PR fixes** 🔨

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by WanderingHogan — APPROVED** [review]

File: components/views/user/User.vue [hunk 1]
```diff
@@ -115,10 +114,6 @@ export default Vue.extend({
       })
     })
   },
-  beforeDestroy() {
-    clearInterval(this.$data.timestampRefreshInterval)
-    this.$store.commit('ui/toggleContextMenu', false)
```
Comment (WanderingHogan) [inline components/views/user/User.vue:120]: we should totally take this out of state. I cant think of why we would want this to be stored there in the first place

Comment (WanderingHogan) [reply components/views/user/User.vue:120]: but we can do it later when there is more time, or an overall state cleanup task if you think it makes sense

Comment (josephmcg) [reply components/views/user/User.vue:120]: now i remember why we added the context menu mutation. it was to handle bugs related to removing friends and clicking the context menu mid operation. will fix

**Comment by WanderingHogan** [comment]
still ready for QA after you put the beforeDestroy toggleContextMenu = false back

**Review by stavares843 — APPROVED** [review]

**Pull request merged**